test/Microsoft.ML.Functional.Tests/Debugging.cs

```diff
@@ -158,6 +158,8 @@ public void InspectSchemaUponLoadingData()
         /// Debugging: The progress of training can be accessed.
         /// </summary>
         [Fact]
+        //Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined
+        [Trait("Category", "SkipInCI")]
```

Comment:
I have a tentative fix out for this. Can we keep this enabled until my PR is complete?

Reply:
sure

---
In reply to: [421106644](https://github.com/dotnet/machinelearning/pull/5099#discussion_r421106644) [](ancestors = 421106644)